libfwupd/fwupd-request.c

```diff
@@ -403,6 +457,100 @@ fwupd_request_set_kind(FwupdRequest *self, FwupdRequestKind kind)
 	g_object_notify(G_OBJECT(self), "kind");
 }
 
+/**
+ * fwupd_request_get_flags:
+ * @self: a #FwupdRequest
+ *
+ * Gets the request flags.
+ *
+ * Returns: request flags, or 0 if unset
+ *
+ * Since: 1.8.6
+ **/
+FwupdRequestFlags
+fwupd_request_get_flags(FwupdRequest *self)
+{
+	FwupdRequestPrivate *priv = GET_PRIVATE(self);
+	g_return_val_if_fail(FWUPD_IS_REQUEST(self), 0);
+	return priv->flags;
+}
+
+/**
+ * fwupd_request_set_flags:
+ * @self: a #FwupdRequest
+ * @flags: request flags, e.g. %FWUPD_REQUEST_FLAG_FORCE_GENERIC
```

Comment:
This flag `FWUPD_REQUEST_FLAG_FORCE_GENERIC` doesn't exist in the PR

Reply:
Ta, fixed.